app/connectors/optout/OptOutUpdateRequestModel.scala

```diff
@@ -14,16 +14,16 @@
  * limitations under the License.
  */
 
-package models.optOut
+package connectors.optout
 
 import play.api.libs.json.{Format, Json}
 import play.mvc.Http.Status.{INTERNAL_SERVER_ERROR, NO_CONTENT}
 
 object OptOutUpdateRequestModel {
 
-  private val defaultUpdateReason: Int = 10
+  val itsaOptOutUpdateReason: Int = 10
```

Comment:
Can we name this update reason from API spec

Reply:
I renamed it to optOutUpdateReason this is not the same as api field but it's appropriate because we will later have optInUpdateReason